sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -0,0 +1,88 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Module of Interactive Beam features that can be used in notebook.
+
+The purpose of the module is to reduce the learning curve of Interactive Beam
+users, provide a single place for importing and add sugar syntax for all
+Interactive Beam components. It gives users capability to interact with existing
+environment/session/context for Interactive Beam and visualize PCollections as
+bounded dataset. In the meantime, it hides the interactivity implementation
+from users so that users can focus on developing Beam pipeline without worrying
+about how hidden states in the interactive session are managed.
+
+Note: Backward-compatibility of Interactive Beam is only guaranteed within this
+module. Please only invoke interfaces provided by this module in your notebook
+or application code if you want backward-compatibility.
+"""
+
+from apache_beam.runners.interactive import interactive_environment as ie
+
+
+def watch(watchable):
+  """Watches a watchable so that Interactive Beam understands your pipeline.
+
+  If you write Beam pipeline in a notebook or __main__ module directly, since
```

Comment:
For a follow up PR, I will suggest working with @rosetn to improve this comment. This comment will serve as documentation, and it will help to get it reviewed by tech writers.

Reply:
Sure, I'll contact @rosetn directly to work on improve the documentation. Would you please share me their ldap or work email address so that I can reach out?

Reply:
Thx for sharing the contact!
Will wait for @rosetn 's feedback to improve the comments before merging. ETA by this Friday.